app/models/vm_or_template.rb

```diff
@@ -1193,7 +1193,7 @@ def refresh_on_reconfig
     unless ext_management_system.authentication_status_ok?
       raise _("%{table} failed last authentication check") % {:table => ui_lookup(:table => "ext_management_systems")}
     end
-    EmsRefresh.reconfig_refresh(self)
+    EmsRefresh.refresh(self)
```

Comment:
Offline clarify with @agrare that this was done in @lfu's PR already.
This enclosing function is also a dead code. Removing it altogether as well. Good find.